### ISSUE TITLE
Slight issue with docs

### ISSUE THREAD

**Issue by ryantheleach** [Alvinn8/praeter#2]
`"IdeaProjects\praeter\praeter-plugin\build\libs\praeter-plugin-0.1-SNAPSHOT.jar"`

preater-plugin was found inside praeter-plugin not praeter core.

Additionally, needed to add mavenLocal to repositories.
```
repositories {
    mavenLocal()
 }
 ```
 
 It says to make an assets folder, but is it in the root project, or inside resources? This would be handy, as would the examples as plugin form, either in their own repository, or a sub folder in this one. I haven't worked it out yet, but I'm getting there.

**Comment by Alvinn8** [comment]
You're right that the docs weren't very clear there, thanks for the feedback!  The assets folder should be inside the `src/main/resources` folder. Were you able to get it all to work? I do have an example plugin / testing plugin that I use to test features, I can send the source code here for now so you can check how to do stuff, but I agree that adding an example plugin here or in another repo would help people understand how to use the project.
[praeter-test.zip](https://github.com/user-attachments/files/17375570/praeter-test.zip)

**Comment by ryantheleach** [comment]
Thanks, I wasn't expecting such a fast response.

I've been deep in learning best practices on shading, and stumbled down a rabbit hole, so been a little distracted, so the response was timely enough that I hadn't solved it yet!

I'll let you know how I go, but I don't want to be a burden.

**Comment by Alvinn8** [comment]
Cool! Don't worry about being a burden, feel free to ask if you have more questions! I only think it's interesting to hear that people are finding the project and think it's interesting :)
